libraries/botbuilder-dialogs-adaptive/package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "@microsoft/recognizers-text-suite": "1.1.4",
     "adaptive-expressions": "4.1.6",
+    "botbuilder": "4.1.6",
```

Comment:
If we want to introduce dependency to `botbuilder` in `botbuilder-dialogs-adaptive`, can we just replace all imports from `botbuilder-core` with `botbuilder`?

Reply:
I suspect the things I'm importing exist in the core, I'll try to resolve that without adding a `botbuilder` dependency.

Reply:
`EventFactory` is not exported from core, so I went ahead and pointed everything to `botbuilder`.

Reply:
@chon219 to answer

> If we want to introduce dependency to `botbuilder` in `botbuilder-dialogs-adaptive`, can we just replace all imports from `botbuilder-core` with `botbuilder`?

@chon219 yes, the imports can be updated to use `botbuilder` instead of `botbuilder-core`. Then `botbuilder-core` can be removed as a dependency.